src/DIRAC/Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -239,6 +249,19 @@ def _executeCondorCommand(self, cmd, keepTokenFile=False):
         htcEnv = {
             "_CONDOR_SEC_CLIENT_AUTHENTICATION_METHODS": "GSI",
         }
+
+        if self.useSSLSubmission:
+            htcEnv = {
+                "_condor_SEC_CLIENT_AUTHENTICATION_METHODS": "SSL",
+                "_condor_AUTH_SSL_CLIENT_CERTFILE": getCertificateAndKeyLocation()[0],
```

Comment:
What if `getCertificateAndKeyLocation()` returns `False` instead of a tuple? Shouldn't we return `S_ERROR`?
Same question for `getCAsLocation()`